backend/src/services/v2/smtp/templates/releaseReviewRequest.ts

```diff
@@ -0,0 +1,62 @@
+import dedent from 'dedent-js'
+import mjml2html from 'mjml'
+
+import { ReviewKindKeys } from '../../../../types/v2/enums.js'
+import { BaseEmailTemplate, IEmailTemplate } from './baseEmailTemplate.js'
+
+export class ReleaseReviewRequest extends BaseEmailTemplate implements IEmailTemplate {
+  getSubject(resourceName: string): string {
+    return dedent(`
+    You have been requested to review '${resourceName}' on Bailo
+  `)
+  }
+
+  getBody(releaseName: string, reviewKind: ReviewKindKeys, modelId: string, baseUrl: string, author: string) {
```

Comment:
This should probably be called `getText`, as it's the text mime.  Both this and the HTML are technically the body.